.travis.yml

```diff
@@ -25,6 +25,8 @@ jobs:
       script:
         - |
           docker build --tag quay.io/open-cluster-management/build-harness:latest .
-          docker login quay.io/open-cluster-management -u ${DOCKER_USER} -p ${DOCKER_PASS}
-          docker push quay.io/open-cluster-management/build-harness:latest
+          if [ "$TRAVIS_REPO_SLUG" == "open-cluster-management/build-harness-extensions" ]; then
```

Comment:
Can you add an additional condition here for `TRAVIS_EVENT_TYPE ` is not "pull_request". That will prevent it from causing failures in PRs.

Reply:
k

Reply:
Done